examples/05 Introduction to Libraries.ipynb

```diff
@@ -0,0 +1,382 @@
+      "text/plain": [
+       "<Diagram '[CDB] Library Product'>"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "diagram = model.diagrams.by_name('[CDB] Library Product')\n",
+    "diagram"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note here again: the class of the library has to be created as a REC and a RPL of the REC has to be instantiated in the project. Otherwise, we wouldn't view it beneath."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Product\n"
+     ]
+    }
+   ],
+   "source": [
+    "for component in model.oa.all_classes:\n",
+    "    print(component.name)"
```

Comment:
This seems very redundant with lines 143f. What's the difference here, except that one addresses `model.la` and the other `model.oa`? If there is none, we can just drop this cell.